verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.js

```diff
@@ -0,0 +1,9 @@
+describe('Linelist table', function () {
```

Comment:
cypress apparently ships with typescript support, can we stick to typescript and avoid javascript please?

https://docs.cypress.io/guides/tooling/typescript-support.html#Install-TypeScript

Reply:
Ah, thanks for catching, just an oversight. Fixed in https://github.com/open-covid-data/healthmap-gdo-temp/pull/158